tower-layer/src/lib.rs

```diff
@@ -14,7 +14,7 @@
 //!
 //! A middleware implements the [`Layer`] and [`Service`] trait.
 //!
-//! [`Service`]: https://docs.rs/tower/trait.Service.html
+//! [`Service`]: https://docs.rs/tower/*/tower/trait.Service.html
```

Comment:
Thanks for fixing this! Can these also be switched to intradoc links? I think the following should work:

```
//! [`Service`]: tower_service::Service
```